packages/backend-common/package.json

```diff
@@ -75,9 +76,11 @@
     "node-abort-controller": "^3.0.1",
     "node-fetch": "^2.6.7",
     "raw-body": "^2.4.1",
+    "request": "^2.88.2",
```

Comment:
Is it possible to stick to `node-fetch` rather than `request`? It's what we use in other places and is a bit more future proof with our planned future setup

Reply:
The kubernetes-client library is about to switch to node-fetch! https://github.com/kubernetes-client/javascript/blob/master/FETCH_MIGRATION.md

We just need to wait till 1.x.x are released

We can keep request temporarly or wait before we release this.